apis/swagger.yml

```diff
@@ -1438,6 +1387,81 @@ definitions:
         description: "Maximum IO in bytes per second for the container system drive (Windows only)"
         type: "integer"
         format: "uint64"
+      # Applicable to co-location kernel
+      BlkFileLevelSwitch:
+         description: "Turn on/off file-level throttle"
+         type: "integer"
+         format: "int64"
```

Comment:
Maybe we can change this to type boolean in the future. Not blocking, just record this.